docs/planning/timeline-editing.mdx

```diff
@@ -40,40 +40,94 @@ Click on the plus button to add a new row to the timeline. A row will be added b
+<br />
+<br />
 
-:::info Note
-Use of Auto mode can result in degraded Aerie UI performance when visualizing thousands of activities on the timeline. It is recommended that you use a smaller, manually specified height for this scenario. Alternatively you can filter the activities down to a more manageable set to improve performance.
+:::info
+Note All activity layers within a row are drawn on the same canvas so that all of the activities are grouped and packed together correctly. Activities will take on the color of the layer that they came from.
 :::
 
-Below is a demonstration of this workflow.
+### Manage how Activities are Rendered
 
-<video controls>
-  <source src={timelineActivityHeightEditingDemo} />
-</video>
+An Activity Options section will appear in the row editor when the row has at least one activity layer. Activity options govern how activities are rendered for this row.
+
+<figure>
+  <img
+    alt="Aerie UI - Timeline Activity Options"
+    src={timelineActivityOptionsEditor}
+    style={{ maxHeight: '100%', width: '100%' }}
+  />
+  <figcaption>Figure 3: Aerie UI Timeline Activity Options Editor - Manage Activity Rendering</figcaption>
+</figure>
+
+#### 1. Activity Display Mode
+
+Activities can be rendered in two ways:
+
+- `Grouped Mode`: Activities are recursively grouped by type. In this mode, activity groups appear in the left side of the row and can be expanded to reveal the directives and spans within each type group. If a directive/span has descendents, the directive/span can be expanded to reveal its grouped descendents. This behavior continues down the decomposition tree until the resulting directive/span has no descendents. This mode is useful for understanding larger sets of activities in a broad sense across an entire plan, especially if the plan is long or contains a significant number of activities that cannot easily be visualized in a small area.
+- `Compact Mode`: Activities from all of the layers are packed together in a space efficient waterfall manner. This mode is useful for visualizing smaller sets of activities in a more traditional timeline view. In this mode the row will not autosize and must be manually adjusted to accommodate the current activity packing solution. This allows for a more precise adjustment of the density in the timeline.
+
+#### 2. Activity Label Visibility
+
+There are several options to control how activity labels appear in the timeline:
+
+- `On`: Always show activity labels regardless of collision.
+- `Off`: Never show activity labels.
+- `Auto`: Show activity labels when they do not collide with other labels or activities.
+
+#### 3. Directive and Span Visibility
+
+By default, both directives and spans are shown in a row. Alternatively you may choose to show only directive or only spans.
+
+#### 4. Hierarchy Root
```

Comment:
Not a comment on the docs, but something this made me think about: Does this option do anything in `Compact Mode`? Is it even visible in `Compact Mode`?

Reply:
`Hierarchy Root` option is only visible in `Grouped Mode`

Reply:
Got it. In that case, here's a possible suggestion that breaks up the paragraph (feel free to rework/reject):

```
#### 4. Hierarchy Root

_This option is only available in `Grouped` mode._

Affects which activities appear in a type group. By default, `Flat` is selected.

- `Flat`: Both activity directives and spans will appear in the topmost group. This is useful when examining all activities resulting from a simulation without needing to consider activity decomposition.
- `By Directive`: Only activity directives will appear in the topmost group, effectively hiding any child spans until further down in the trees. This is useful when examining the timeline from a decomposition perspective.
```

It looks like so when rendered:
<img width="873" alt="image" src="https://github.com/NASA-AMMOS/aerie-docs/assets/62574120/2c9196cc-bdd5-497d-a024-3117921134f6">